pyanaconda/bootloader.py

```diff
@@ -1614,6 +1615,14 @@ def write_config(self):
             if rc:
                 log.error("failed to set default menu entry to %s", productName)
 
+        # set menu_auto_hide grubenv variable if we should enable menu_auto_hide
+        # set boot_success so that the menu is hidden on the boot after install
+        if self.menu_auto_hide:
+            rc = util.execInSysroot("grub2-editenv",
+                            ["-", "set", "menu_auto_hide=1", "boot_success=1"])
+            if rc:
+                log.error("failed to set menu_auto_hide=1")
```

Comment:
Also there is a question if this should be fatal error. In other words if we shouldn't raise an exception?

Reply:
If this fails everything will still work fine, except that the grub2 bootmenu will show everyboot, as it was in F28 and before. So this is not really fatal. I based not making this fatal on the code which selects the default boot menu entry also simply logging an error, rather then throwing an exception.